synology-letsencrypt.sh

```diff
@@ -52,7 +71,6 @@ fi
     --key-type "rsa4096" \
     --email "$EMAIL" \
     --dns "$DNS_PROVIDER" \
+    --dns.resolvers=8.8.8.8,8.8.4.4 \
```

Comment:
Is it necessary to hardcode the DNS?

lego's default is:
"The default is to use the system resolvers, or Google's DNS resolvers if the system's cannot be determined."

Reply:
Urk. No, that's an oversight; those are from my local fork in other ways. I'll pull that out.